src/client.cpp

```diff
@@ -549,6 +580,29 @@ namespace quicr {
                 }
                 return true;
             }
+            case messages::ControlMessageType::kPublish: {
+                messages::Publish msg([](messages::Publish& msg) {
+                    if (msg.content_exists) {
+                        msg.group_0 =
+                          std::make_optional<messages::Publish::Group_0>(messages::Publish::Group_0{ 0, 0 });
+                    }
+                });
+                msg_bytes >> msg;
+
+                const FullTrackName tfn{ .name_space = msg.track_namespace, .name = msg.track_name };
+                const TrackHash th(tfn);
```

Comment:
Params are not being parsed and passed to the app.  For example, dynamic groups param.